packages/combo-box/test/basic.test.js

```diff
@@ -207,10 +208,6 @@ describe('Properties', () => {
       expect(comboBox.hasAttribute('focused')).to.be.false;
     });
 
-    it('should not throw on focusout', () => {
-      expect(() => comboBox.dispatchEvent(new Event('focusout'))).not.to.throw(Error);
-    });
```

Comment:
This test was added in #750 when we changed the logic to postpone overlay rendering. That logic was later reverted in https://github.com/vaadin/web-components/pull/2467 so this test became meaningless, I just missed to remove it.